apps/application-status/enums/dropdownEnum.ts

```diff
@@ -0,0 +1,8 @@
+enum Zenhub {
+  ENTITIES = '5bf2f2164b5806bc2bf60531',
```

Comment:
what are these keys? should they be in a secret?

Reply:
I thought the Zenhub's boardID is a public ID because it is exposed in the URL link https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/ so it can be enum like that. I also tried putting it in env file, but when pushing it, it would fail some CI checkings